src/views/citation_editable.ts

```diff
@@ -59,15 +63,27 @@ export class CitationEditableView extends CitationView<EditableBlockProps> {
     )
   }
 
+  public eventHandlers = () => {
+    this.dom.addEventListener('mouseup', this.handlcick, { capture: true })
+  }
+  public handlcick = (event: MouseEvent) => {
+    if (
+      this.can.seeReferencesButtons &&
+      !isDeleted(this.node) &&
+      event.button === 0
+    ) {
+      const attrs = getActualAttrs(this.node) as CitationAttrs
+      if (attrs.rids.length) {
```

Comment:
is it actually possible to click on citation that has no rids?

Reply:
yes, its possible.